aiopg/connection.py

```diff
@@ -806,6 +812,7 @@ def _ready(weak_self: "weakref.ref[Any]") -> None:
                     # chain exception otherwise
                     exc2.__cause__ = exc
                     exc = exc2
+            self.notifies.close(exc)
```

Comment:
What do you think? Is it important here to include an exception info from psycopg to the ClosableQueue? Instead we could raise QueueIsClosedError or smth like this.

Reply:
It's just a question, I'm ok with current implementation.

Reply:
> What do you think? Is it important here to include an exception from psycopg to the ClosableQueue? Instead we could raise QueueIsClosedError or smth like this.

My feeling is that there might be some information in the psycopg2 exception that may be useful to the application to determine _why_ the connection got broken?  I think it's slightly more useful to keep the original exception.  I am leaning towards keeping as is.

Reply:
I am choosing to pass the original exception as is.